src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -59,7 +59,7 @@
         "symfony/scheduler": "^6.4.4|^7.0.4",
         "symfony/security-bundle": "^6.4|^7.0",
         "symfony/semaphore": "^6.4|^7.0",
-        "symfony/serializer": "^6.4|^7.0",
+        "symfony/serializer": "^7.1",
```

Comment:
Why is this change needed?

Reply:
The test suite fails if serializer is < 7.1: https://github.com/symfony/symfony/actions/runs/11384686629/job/31672929730?pr=58589#step:8:11669

Reply:
I wonder why this didn't fail before though. 🤔

Reply:
Maybe the behavior in ApiAttributeTests (switching the assertion based on the HttpKernel version being used) was not actually dependent only in the HttpKernel but also on the Serializer version, but no CI job was installing a recent HttpKernel with an old Serializer